MovieViewer/MoviesSearchViewController.swift

```diff
@@ -189,9 +194,71 @@ extension MoviesSearchViewController: UITableViewDataSource {
         let movie = filteredMovies![indexPath.row]
         cell.setData(movie)
         cell.setTheme()
+        if let id = movie.id {
```

Comment:
Just pointing out, this code might belong in `setData` or `setTheme`, but those files don't have access to the `favorited` data. So ... okay, it can stay here. :)